frontend/components/NavigationBar.vue

```diff
@@ -15,12 +15,12 @@
               <a
                 href="#"
                 class="bg-gray-900 text-white px-3 py-2 rounded-md text-sm font-medium"
-                >Mapa</a
-              >
+                >{{ $t("NavigationBar.map") }}
+              </a>
               <a
                 href="#"
                 class="text-gray-300 hover:bg-gray-700 hover:text-white px-3 py-2 rounded-md text-sm font-medium"
-                >Dodaj firmę</a
+                >{{ $t("NavigationBar.add-company") }}</a
```

Comment:
Can we unify this to form example `snake_case`? For now we have `PascalCase` and `kebab-case` and it's not comfortable to read this.